13:54:35,555 rasa_core.processor DEBUG Received user message 'Hi' with act '{'name': 'greet', 'confidence': 0.9653160257870657}' and slots '[{'start': 0, 'end': 0, 'value': None, 'entity': 'password_reset', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'password_reset'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'O', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'O'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'application_skype', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'application_skype'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'application_salesforce', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'application_salesforce'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'application_omniview', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'application_omniview'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'ID', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'ID'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'number', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'number'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'security_code', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'security_code'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'request_number', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'request_number'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'finished', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'finished'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'access', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'access'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'machine', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'machine'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'send_otp', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'send_otp'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'approval', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'approval'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'admin_access', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'admin_access'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'login_able', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'login_able'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'email_sent', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'email_sent'}]'
13:54:35,555 rasa_core.processor DEBUG Currently not implemented: _log_slots
13:54:35,576 rasa_core.featurizers DEBUG Feature 'nlu_slot_O' (value: '0') could not be found in feature map. Make sure you added all acts and nlu_slots to the domain
13:54:35,590 rasa_core.policies.ensemble DEBUG Predicted next action using policy_0_KerasIDarePolicy
13:54:35,613 rasa_core.featurizers DEBUG Feature 'action_slot_O' (value: '0') could not be found in feature map. Make sure you added all acts and nlu_slots to the domain
14:30:45,909 rasa_core.processor DEBUG Received user message 'hi' with act '{'name': 'greet', 'confidence': 0.9653160257870657}' and slots '[{'start': 0, 'end': 0, 'value': None, 'entity': 'password_reset', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'password_reset'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'O', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'O'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'application_skype', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'application_skype'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'application_salesforce', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'application_salesforce'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'application_omniview', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'application_omniview'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'ID', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'ID'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'number', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'number'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'security_code', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'security_code'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'request_number', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'request_number'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'finished', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'finished'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'access', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'access'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'machine', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'machine'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'send_otp', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'send_otp'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'approval', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'approval'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'admin_access', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'admin_access'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'login_able', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'login_able'}, {'start': 0, 'end': 0, 'value': None, 'entity': 'email_sent', 'confidence': 0, 'extractor': 'ner_crf', 'slot': 'email_sent'}]'
14:30:45,909 rasa_core.processor DEBUG Currently not implemented: _log_slots
14:30:45,918 rasa_core.featurizers DEBUG Feature 'nlu_slot_O' (value: '0') could not be found in feature map. Make sure you added all acts and nlu_slots to the domain
14:30:45,924 rasa_core.policies.ensemble DEBUG Predicted next action using policy_0_KerasIDarePolicy
14:30:45,947 rasa_core.featurizers DEBUG Feature 'action_slot_O' (value: '0') could not be found in feature map. Make sure you added all acts and nlu_slots to the domain
16:41:44,894 rasa_core.tracker_store INFO Retrieve begin 2018-11-14 16:41:44.893976
16:41:44,910 rasa_core.tracker_store INFO Retrieve end 2018-11-14 16:41:44.910822
16:41:44,913 rasa_core.utils ERROR Traceback (most recent call last):
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/utils.py", line 467, in greenlet_wrapper
    function(*arguments)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/processor.py", line 90, in handle_channel_asynchronous
    self.handle_message(message, usr_channels, ha_channel)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/processor.py", line 101, in handle_message
    tracker, incoming_ids = self._get_tracker(message.sender_id)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/processor.py", line 523, in _get_tracker
    tracker = self.tracker_store.get_or_create_tracker(usr_id, ha_id)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/tracker_store.py", line 28, in get_or_create_tracker
    tracker = self.retrieve(usr_id)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/tracker_store.py", line 135, in retrieve
    return self.deserialise_tracker(sender_id, stored)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/tracker_store.py", line 71, in deserialise_tracker
    tracker.recreate_from_dialogue(dialogue)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/trackers.py", line 264, in recreate_from_dialogue
    self.replay_events()
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/trackers.py", line 249, in replay_events
    event.apply_to(self)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/events/__init__.py", line 226, in apply_to
    tracker.update_belief_state(self)
  File "/home/arusia/Demo/env/lib/python3.6/site-packages/rasa_core-1.01.m2-py3.6.egg/rasa_core/trackers.py", line 496, in update_belief_state
    ''.format(sk,state))
Exception:  Unknown key area in the state: {'ID': <TextSlot(ID: None)>, 'access': <BooleanSlot(access: None)>, 'admin_access': <BooleanSlot(admin_access: None)>, 'application_omniview': <BooleanSlot(application_omniview: None)>, 'application_salesforce': <BooleanSlot(application_salesforce: None)>, 'application_serviceacc': <BooleanSlot(application_serviceacc: None)>, 'application_skype': <BooleanSlot(application_skype: None)>, 'approval': <BooleanSlot(approval: None)>, 'convert': <BooleanSlot(convert: None)>, 'email_id': <TextSlot(email_id: None)>, 'email_sent': <BooleanSlot(email_sent: None)>, 'feedback': <BooleanSlot(feedback: None)>, 'finished': <BooleanSlot(finished: None)>, 'howto': <BooleanSlot(howto: None)>, 'issue': <BooleanSlot(issue: None)>, 'login': <BooleanSlot(login: None)>, 'login_able': <BooleanSlot(login_able: None)>, 'machine': <TextSlot(machine: None)>, 'name': <TextSlot(name: None)>, 'number': <TextSlot(number: None)>, 'password_reset': <TextSlot(password_reset: None)>, 'pdf_converter': <TextSlot(pdf_converter: None)>, 'request_number': <TextSlot(request_number: None)>, 'resolved': <BooleanSlot(resolved: None)>, 'security_code': <TextSlot(security_code: None)>, 'send_otp': <BooleanSlot(send_otp: None)>, 'sentiment': <BooleanSlot(sentiment: None)>, 'unavailable_slots': <CategoricalSlot(unavailable_slots: None)>, 'wait': <BooleanSlot(wait: None)>, 'xls2pdf': <BooleanSlot(xls2pdf: None)>, 'xls2pdf_harddrive': <BooleanSlot(xls2pdf_harddrive: None)>}

